Library/Homebrew/test/formulary_spec.rb

```diff
@@ -136,6 +136,12 @@ class Wrong#{described_class.class_s(formula_name)} < Formula
     end
 
     context "with installed Formula" do
+      before do
+        allow(Formulary).to receive(:loader_for).and_call_original
+        stub_formula_loader stub_formula("patchelf") { url "patchelf-1.0" }
```

Comment:
If I use `formula("patchelf")` rather than `stub_formula("patchelf")` I get the error message
```
     Failure/Error: stub_formula_loader formula("patchelf") { url "patchelf-1.0" }
     
     ArgumentError:
       wrong number of arguments (given 1, expected 0)
     # ./test/formulary_spec.rb:141:in `block (4 levels) in <top (required)>'
```